FirebaseAdmin/FirebaseAdmin.IntegrationTests/FirebaseAuthTest.cs

```diff
@@ -510,6 +510,115 @@ public async Task ListUsers()
             }
         }
 
+        [Fact]
+        public async Task ImportUsersBasic()
+        {
+            var randomUser = RandomUser.Create();
+            var args = new ImportUserRecordArgs()
+            {
+                Uid = randomUser.Uid,
+                Email = randomUser.Email,
+                DisplayName = "Random User",
+                PhotoUrl = "https://example.com/photo.png",
+                EmailVerified = true,
+            };
+
+            var options = new UserImportOptions();
+            IEnumerable<ImportUserRecordArgs> usersLst = new List<ImportUserRecordArgs>();
+            usersLst = usersLst.Append(args);
+
+            var resp = await FirebaseAuth.DefaultInstance.ImportUsersAsync(usersLst);
+
+            try
+            {
+                Assert.Equal(1, resp.SuccessCount);
+                Assert.Equal(0, resp.FailureCount);
+            }
+            finally
+            {
+                await FirebaseAuth.DefaultInstance.DeleteUserAsync(randomUser.Uid);
+            }
+        }
+
+        [Fact]
+        public async Task ImportUsersPasswordNoHash()
+        {
+            var randomUser = RandomUser.Create();
+            var args = new ImportUserRecordArgs()
+            {
+                Uid = randomUser.Uid,
+                Email = randomUser.Email,
+                PasswordSalt = Encoding.ASCII.GetBytes("abc"),
+                PasswordHash = Encoding.ASCII.GetBytes("def"),
+            };
+
+            var options = new UserImportOptions();
+            IEnumerable<ImportUserRecordArgs> usersLst = new List<ImportUserRecordArgs>();
+            usersLst = usersLst.Append(args);
+            await Assert.ThrowsAsync<ArgumentNullException>(
+                async () => await FirebaseAuth.DefaultInstance.ImportUsersAsync(usersLst, options));
+        }
+
+        [Fact]
+        public async Task ImportUsersPassword()
```

Comment:
I don't think this test case is exactly correct (it's importing a user with incorrect password hashes). Ideally we need something similar to what's in Java. That is import users with a valid Scrypt password hash, and demonstrate that we can login as that user. But feel free to do that change in a future PR. This is acceptable for now.

Reply:
I will do a separate PR follow this